src/layouting/transformation.jl

```diff
@@ -195,6 +195,16 @@ transformationmatrix(x) = transformation(x).model
 transform_func(x) = transform_func_obs(x)[]
 transform_func_obs(x) = transformation(x).transform_func
 
+"""
+    apply_transform(f, data, space)
+Apply the data transform func to the data if the space matches one
+of the the transformation spaces (currently only :data is transformed)
+"""
+apply_transform(f, data, space) = space == :data ? apply_transform(f, data) : data  
```

Comment:
Just to point it out - this might need to be `copy(data)` if there are any mutating operations done on the output. I don't think there are, since we generally go through observables which create copies, but I thought I'd mention it anyway.

Reply:
The identity transforms make a point of returning `data` (or `x`) directly (and verifies that by checking `===` in one of the test cases), so that behavior was mirrored here. I would think any issues with mutating the data would quickly show up in that regard. Let me know if you think this needs any changes.

Reply:
Yeah, I did that at the time because I thought the overhead was worse. And because I didn't see the need for mutation afterwards, but still one should be careful

Reply:
Is there a change (documentation note maybe?) or test you would like to insert here to make sure this stays consistent in the future?